node/cmd/ccq/utils.go

```diff
@@ -197,3 +221,42 @@ func validateRequest(logger *zap.Logger, perms Permissions, apiKey string, qr *g
 
 	return nil
 }
+
+const (
+	GuardianKeyArmoredBlock = "WORMHOLE GUARDIAN PRIVATE KEY"
```

Comment:
Ideally this would be something like `CCQ SERVER PRIVATE KEY`

Reply:
By using the above string we can generate a key by using the following existing command as follows:
`guardiand keygen <someKeyFileName`

Reply:
Could we add a `--ccq` flag to guardiand keygen that simply changes that as @evan-gray suggested? Is it even worth it?

Reply:
Changed it to use a block type of "CCQ SERVER SIGNING KEY".